staging/src/k8s.io/api/networking/v1beta1/types.go

```diff
@@ -220,9 +220,9 @@ type IngressRule struct {
 type IngressRuleValue struct {
 	//TODO:
 	// 1. Consider renaming this resource and the associated rules so they
-	// aren't tied to Ingress. They can be used to route intra-cluster traffic.
+	//    aren't tied to Ingress. They can be used to route intra-cluster traffic.
 	// 2. Consider adding fields for ingress-type specific global options
-	// usable by a loadbalancer, like http keep-alive.
+	//    usable by a loadbalancer, like http keep-alive.
```

Comment:
What happens if a contributor does not put leading spaces to align list elements as you do here? Does previously working comment now look bad?